packages/blade/private/server/worker/tree.ts

```diff
@@ -479,6 +551,7 @@ const renderReactTree = async (
     if (options.errorReason) url.searchParams.set('reason', options.errorReason);
   }
 
+  const sessionId = requestHeaders.get('X-Session-Id');
```

Comment:
```suggestion
  // The ID of the browser session.
  const sessionId = requestHeaders.get('X-Session-Id');

```

Let's keep the comment?